.github/workflows/push.yml

```diff
@@ -39,4 +39,4 @@ jobs:
             docker pull pyronear/pyro-platform:latest
             cd pyro-platform-prod && docker compose down && docker compose up -d
             docker inspect -f '{{ .Created }}' $(docker compose images -q frontend)
```

Comment:
I'm not sure of the relevance of this then :/

We could delete it as well I guess